yarn.lock

```diff
@@ -8407,7 +8407,7 @@ react-is@^17.0.1, react-is@^17.0.2:
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-17.0.2.tgz#e691d4a8e9c789365655539ab372762b0efb54f0"
   integrity sha512-w2GsyukL62IJnlaff/nRegPQR94C/XXamvMWmSHRJ4y7Ts/4ocGRmTHvOs8PSE6pB3dWOrD/nueuU5sduBsQ4w==
 
-react-pdf@^6.2.2:
+react-pdf@^6.2.0:
   version "6.2.2"
```

Comment:
:point_up_2:  It still uses `6.2.2`, so the change in `package.json` could be reverted.

Reply:
🤦‍♀️ Of course.. Will "officially" user the locked version then in the `package.json`.

Reply:
Done!